### PR TITLE
small fixes and additions

### PR DESCRIPTION
**Pull request by skatans**

### CODE REVIEW

File: install.html [hunk 1]
```diff
@@ -74,11 +74,6 @@
                     <p>This page helps you to choose and install a hypervisor into your personal computer.</p>
                 </article>
 
-                <details class="showme">
-                    <summary>Show me what it looks like...</summary>
-                    and this is the bulk...  blah blah
-                </details>
-
```
Comment (Jasata) [inline install.html]: This is here waiting for a few style changes and then to be taken into use elsewhere. Won't be removed for now.

File: install.html [hunk 2]
```diff
@@ -299,7 +294,7 @@ <h1 id="install"><i class="fa fa-wrench"></i>Installing the Hypervisor</h1>
                                 <li>Navigate to the <a target="_blank" href="https://www.virtualbox.org/wiki/Linux_Downloads">Linux specific download page</a>.</li>
                                 <li>Follow the instructions therein to install.</li>
                                 <li>Navigate to <a target="_blank" href="https://www.virtualbox.org/wiki/Downloads">generic download page</a> and download VirtualBox Extension Pack</li>
-                                <li>TBA (install extension pack)</li>
+                                <li>After downloading, double-click on the Extension Pack to install.</li>
```
Comment (Jasata) [inline install.html]: Are you sure that double-clicking the extension pack works in a **LInux** OS? If you really are, then this is OK.

File: install.html [hunk 3]
```diff
@@ -296,10 +296,14 @@ <h1 id="install"><i class="fa fa-wrench"></i>Installing the Hypervisor</h1>
...
                                 <li>After downloading, double-click on the Extension Pack to install.</li>
                             </ol>
+                            <p>Issues in downloading or installing Oracle VirtualBox? Visit their <a href="https://www.virtualbox.org/manual/ch02.html#installation_windows">manual</a> for further instructions.</p>
                         </details>
                         <details class="install-hypervisor">
                             <summary>VMware Player & Pro</summary>
-                            VMware Player & Pro
+                            <ol>
+                                <li>VMware Player & Pro</li>
+                            </ol>
```
Comment (Jasata) [inline install.html]: Is this one-item oldered list not redundant title (what already exists as the <summary>? Why is this a list item btw?

**Review by Jasata — CHANGES_REQUESTED** [review]
Please address the review comments.

File: install.html (hunk 1 above)
Comment (skatans) [reply install.html]: Ok! Added this back.

File: install.html (hunk 3 above)
Comment (skatans) [reply install.html]: It was a placeholder for the installation steps, replaced with first actual step now

File: install.html (hunk 2 above)
Comment (skatans) [reply install.html]: Replaced this with instructions on how to browse for the exp pack from virtualbox

**Pull request merged**